grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -32,16 +32,35 @@
     -->
 
     <import resource="client-batch-job-context.xml"/>
+
+    <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
-    <import resource="client-preprocessor-step.xml" />
+    <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
-        <batch:step id="clientPreprocessor" next="clientWorkflowOff">
-            <batch:tasklet ref="clientPreprocessTasklet" transaction-manager="clientTransactionManager"/>
-        </batch:step>
-        <batch:step id="clientWorkflowOff" next="clientJcrNodes">
-            <batch:tasklet ref="clientWorkflowOffTasklet" transaction-manager="clientTransactionManager"/>
+        <!--  Do any presync tasks concurrently that we can, e.g turning off workflows -->
+        <batch:split id="preSyncFlow" task-executor="asyncTaskExecutor" next="clientNamespaceSync">
+            <batch:flow>
+                <batch:step id="clientWorkflowOff">
```

Comment:
Performance benefits are marginal at this point; but any small wins are great, especially since the complexity is managed by Spring Batch.  That said it's more to establish the pattern, that we should be doing independent tasks concurrently.

I hope to move namespaceSync into preFlowSync eventually, but for the sake of keeping the PR small...This will make this change a bit more significant.